### PR TITLE
Final project

### PR DESCRIPTION
**Pull request by SonaliBedge**
Final project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,14 +2,18 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" type="image/jpg" href="./src/media/todo-logo-img.jpg" alt="Logo" width="45px" height="45px" />
+    <!-- <link rel="icon" type="image/svg+xml" href="/vite.svg" /> -->
```
Comment (brandaogabriel7) [inline index.html:6]: Remove this commented out code.

File: src/App.css [hunk 2]
```diff
@@ -17,6 +17,31 @@
 .logo.react:hover {
   filter: drop-shadow(0 0 2em #61dafbaa);
 }
+.nav {
+  /*background-color:  hsl(210, 50%, 92%);/* Background color */
```
Comment (brandaogabriel7) [inline src/App.css:21]: Remove this commented out code.

File: src/components/TodoContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  }, [sortOrder, sortBy]);
+
+  // Show or hide AddTodoForm based on current location
+  useEffect(() => {
+    if (location.pathname === "/TodoContainer") {
+      setShowAddTodoForm(true);
+    } else {
+      setShowAddTodoForm(false);
+    }
+    console.log(showAddTodoForm);
```
Comment (brandaogabriel7) [inline src/components/TodoContainer.jsx:31]: I suppose this log was meant for debugging, do you still need it?

File: src/components/TodoContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+          />
+        </a>
+      </div>
+    </>
+  );
+}
+
+// Define prop types for TodoContainer component
+TodoContainer.propTypes = {
+  addTodo: PropTypes.func,
```
Comment (brandaogabriel7) [inline src/components/TodoContainer.jsx:219]: This prop doesn't exist in your component definition.

File: src/components/TodoContainer.jsx [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+
+  // Show or hide AddTodoForm based on current location
+  useEffect(() => {
+    if (location.pathname === "/TodoContainer") {
+      setShowAddTodoForm(true);
+    } else {
+      setShowAddTodoForm(false);
+    }
+    console.log(showAddTodoForm);
+  }, [location]);
```
Comment (brandaogabriel7) [inline src/components/TodoContainer.jsx:32]: You're missing `showAddTodoForm` as dependency for this `useEffect`.

**Review by brandaogabriel7 — APPROVED** [review]
Your final project looks great and everything is working, Sonali!

Congratulations!

File: src/components/TodoContainer.jsx (hunk 5 above)
Comment (SonaliBedge) [reply src/components/TodoContainer.jsx:32]: Done!

**Pull request merged**